client/blocks/inline-help/inline-help-search-results.jsx

```diff
@@ -15,59 +17,136 @@ import { recordTracksEvent } from 'state/analytics/actions';
 import QueryInlineHelpSearch from 'components/data/query-inline-help-search';
 import PlaceholderLines from './placeholder-lines';
 import { decodeEntities, preventWidows } from 'lib/formatting';
-import getInlineHelpSearchResultsForQuery from 'state/inline-help/selectors/get-inline-help-search-results-for-query';
+import getSearchResultsByQuery from 'state/inline-help/selectors/get-inline-help-search-results-for-query';
 import getSelectedResultIndex from 'state/inline-help/selectors/get-selected-result-index';
+import getInlineHelpCurrentlySelectedResult from 'state/inline-help/selectors/get-inline-help-currently-selected-result';
 import isRequestingInlineHelpSearchResultsForQuery from 'state/inline-help/selectors/is-requesting-inline-help-search-results-for-query';
 import hasInlineHelpAPIResults from 'state/selectors/has-inline-help-api-results';
 import { selectResult } from 'state/inline-help/actions';
 import { localizeUrl } from 'lib/i18n-utils';
+import { SUPPORT_TYPE_ADMIN_SECTION, SUPPORT_TYPE_API_HELP } from './constants';
 
 function HelpSearchResults( {
 	hasAPIResults = false,
 	isSearching = false,
-	openResult,
+	onSelect,
 	searchQuery = '',
-	searchResults,
+	searchResults = [],
+	selectedResult = {},
 	selectedResultIndex = -1,
 	selectSearchResult,
 	translate = identity,
 	placeholderLines,
+	recordTracksEvent,
 } ) {
-	const selectResultHandler = ( selectionIndex ) => ( event ) => {
-		const selectedResult = searchResults?.[ selectionIndex ] ?? null;
-		selectSearchResult( selectionIndex );
-		openResult( event, selectedResult );
+	const supportType = useRef( searchResults?.[ 0 ]?.support_type );
+
+	function getTitleBySectionType( addSection, type ) {
+		if ( ! addSection ) {
+			return null;
+		}
+
+		let title = '';
+		switch ( type ) {
+			case SUPPORT_TYPE_API_HELP:
+				title = translate( 'Support articles:' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'Support articles' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/inline-help-add-admin-sections-ui)